.travis.yml

```diff
@@ -6,7 +6,7 @@ dist: trusty
 python:
   - "2.6"
   - "2.7"
-  - "3.4"
+#  - "3.4"
```

Comment:
Instead of disabling it, add to the `known_failures` section. That way, you'll find out when it's working without having to explicitly go enable it again.